cipherscan

```diff
@@ -250,6 +250,8 @@ Some useful OpenSSL options:
                     that require turning TLS after initial protocol specific
                     hello
 -servername name    Request SNI support for connections
+-proxy proxyhost:port Connect to the scan target via specified proxy
+                    (req. OpenSSL 1.0.x)
```

Comment:
Hmmm, this is interesting. The copy of OpenSSL supplied with cipherscan works with the -proxy option, but a local copy of 1.0.2j doesn't work with it. May have to change that to something like "Requires supplied OpenSSL or >1.1.0"? Trying to figure out exactly where this got introduced to OpenSSL is a challenge because the feature appears to predate their migration to Git (at least, I couldn't find anything about it in their repository).